index.html

```diff
@@ -16,7 +16,50 @@
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <!-- ** Header ** -->
+    <header class="header">
+      <nav class="navbar">
+      <h1 class="title">CodeYourFuture News</h1>
+      <p class="desc" >CYF students are trained in full-stack web development by volunteers from the tech industry, putting a strong emphasis on collaboration and product development through tech projects.
+      </p>
+      </nav>
+    </header>
+     <!-- ** Header ** -->
+      <!-- ** Main ** -->
+    <main class="main">
+      <!-- ** Article ** -->
+      <article>
+      <section class="article">
+        <h2 id="css">CSS Week 1</h2>
+        <img class="articleImg" src="Img/css design.jpg" alt="css design">
+        <p class="articletext">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Iusto excepturi eveniet eius recusandae neque aliquid a delectus itaque dolor dignissimos? Maxime adipisci ratione qui beatae quam provident dolorum facilis amet!</p>
+        <h5>Date: 27 July 2022</h5>
```

Comment:
Ideally you would avoid skipping 'h' (header) levels like this. Although it may make sense from a styling perspective,  semantically, it is more logical that H1 -> H2 -> H3 -> H4 -> H5